dashboard/app/views/levels/_external.html.haml

```diff
@@ -0,0 +1,9 @@
+%link{rel: 'stylesheet', href: asset_path('blockly/css/common.css')}
+
+#external
+  :javascript
+    var options = #{@level.properties['options'].to_json}
+
+    $.ajax("#{asset_path @level.properties['href']}").done(function (data) {
+      $('#external').html(data);
+    });
```

Comment:
Good catch!  I don't think this blocks check-in, but I'll add this to the backlog to fix before this widget goes public.